contrib/release/lib/common.sh

```diff
@@ -357,52 +357,6 @@ common::exit () {
   common::cleanexit $etype
 }
 
-#############################################################################
-# Simple yes/no prompt
-#
-# @optparam default -n(default)/-y/-e (default to n, y or make (e)xplicit)
-# @param message
-common::askyorn () {
```

Comment:
I was about to add a user for this, so we can either keep it in this PR or when I submit the user, issue a partial revert.

Reply:
No worries, I've put this one back in!